c7n/resources/timestream.py

```diff
@@ -0,0 +1,137 @@
+from c7n.manager import resources
+from c7n.actions import Action
+from c7n.query import DescribeSource, QueryResourceManager, TypeInfo
+from c7n.utils import local_session, type_schema
+from c7n.tags import (
+    TagDelayedAction,
+    TagActionFilter,
+    Tag as TagAction,
+    RemoveTag as RemoveTagAction
+)
+
+
+class DescribeTimestream(DescribeSource):
```

Comment:
We could use the RGT api here to read tags for both `timestream-database` and `timestream-table`.

Reply:
see my comment above, basically i'm not convinced that timestream is _fully_ supported with resourcegroupstaggingapi i would note that it also doesnt show up in my tag explorer in the aws console